common.yml

```diff
@@ -0,0 +1,22 @@
+services:
+  # Common configuration for all of our core uservices
+  oada-uservice:
+    build:
+      context: ./oada
+    restart: unless-stopped
+    networks:
+      - startup_net
+      - kafka_net
+      - arango_net
+    volumes:
+      - ./oada-srvc-docker-config.js:/oada-srvc-docker-config.js
+    environment:
+      - NODE_ENV=${NODE_ENV:-development}
+      - DEBUG=${DEBUG:-*}
+      # Should just be an external script someone can run on their instance?
+      - arangodb__ensureDefaults="true"
```

Comment:
this should not be in common, it should only be in startup. None of the other services call init in the arangolib, so it has no effect on them and making it common will hide that.

Reply:
Seems like someone would just do a
```bash
$ arangodb__ensureDefaults=true docker-compose up
```
then?

Reply:
yes, that is all the dev-docker-compose.yml file did originally.  The point was nobody, including me, could remember that variable name :).  Perhaps a better solution is just moving the insertion of default tokens/users into an admin script that someone can run whenever to populate their installation with dummy users/tokens.  I think I'd still like startup to remove them by default to prevent accidentally open installations, but if we move it to a one-time script I'm not sure how to do that reliably: i.e. after you ran the script, the next time startup comes up it will wipe them out again.  That points to using an environmnet variable, but that means startup can just do it like it does now, no need for the script.  I suppose probably the original dev-docker-compose.yml which sets this environment variable on startup is still the best approach.

Reply:
Everything in this repo _is_ dev. You don't start a production deploy with the git repo. You don't have the git repo at all. This PR has not addressed production. GA for pushing images and a place/idea for publishing production compose files is still needed.

Reply:
The point of the repo is to produce a working "prod".  Failure to 100% prevent leakage into the release compose file seems likely if the core compose file has things turned on by default that cannot make it into production.  Shouldn't the docker-compose in dev on a release branch basically be the actual release docker-compose?  Otherwise, we aren't really testing production when we test, we're testing dev and hoping production's release compose file didn't forget an environment variable somewhere.  I assumed that the process here was going to be: dev on things, make a release candidate whose docker-compose is the production docker-compose, test it, then copy the tested docker-compose to the github as the release for that version of OADA?  i.e. running OADA means downloading that compose file and adding your override alongside it.  At least, I think that's my understanding of how you guys want to do it, right?

Reply:
Maybe, but like I said, haven't considered production at all. This effort was on making images and making them not massive -- that's what I had time for. If we can agree on that, then we have a base to work with to figure out releases.

What do you mean exactly by "produce a working prod" ? From my perspective the point of this repo is to produce working images -- those aren't dev or prod. How you make and configure containers makes it dev vs. prod. Seems reasonable that we could get to the point where making a "recommended" (I guess?) prod compose file consists of taking a dev compose file (tracked in this repo), updating the tags to a versioned release of the images, and then posting it somewhere for people to use. I'm not sure what would be left to get to that point.

In this particular case, I think this should just be a script that someone can be run and not some magical env that could accidently trigger the script. I don't see much value in looking for the dev data at startup, because it can't be there on accident unless you added it yourself.